test/test-cases/typescript-css-modules/app/sku.config.js

```diff
@@ -3,5 +3,13 @@ module.exports = {
     client: 'src/client.tsx',
     render: 'src/render.tsx',
     server: 'src/server.tsx'
+  },
+  // Use 'dangerouslySetESLintConfig' so we can
+  // check that overrides are written to disk.
```

Comment:
Is there a corresponding test for the written output?

Reply:
Just added a test 👍

Reply:
Let me know what you think.